.gitattributes

```diff
@@ -1,2 +1,3 @@
 # Force LF on text files (requires git>=2.10)
-* text=auto eol=lf
\ No newline at end of file
+* text=auto eol=lf
+* .gif binary
```

Comment:
I think there is an extra space here that doesn't belong there. The `.gitattributes` syntax is `pattern attr1 attr2 ...`, and the pattern should match all `*.gif` files.